Examples/Detectors/CMakeLists.txt

```diff
@@ -5,5 +5,5 @@ add_subdirectory_if(Geant4Detector ACTS_BUILD_EXAMPLES_GEANT4)
 add_subdirectory(MagneticField)
 add_subdirectory(TGeoDetector)
 add_subdirectory(TelescopeDetector)
-add_subdirectory_if(MuonSpectrometerMockupDetector ACTS_BUILD_EXAMPLES_GEANT4)
-
+add_subdirectory(DetectorInspectors)
```

Comment:
should we do `add_subdirectory_if` here and remove the checks from `Examples/Detectors/DetectorInspectors/CMakeLists.txt`?

Reply:
Not really, because I think we may have different inspectors in the future with different dependencies.

Reply:
E.g. I will remove the direct json dependency probably and split it into a common reader function.